### ISSUE TITLE
Documentation for private mutations

### ISSUE THREAD

**Issue by garfinjm** [nextstrain/nextclade#202]
Hello,

I'm interested to know a little more about how to the private mutation QC check is defined, I pretty regularly have samples that fail that check that otherwise look OK.

Thanks again for making such a great tool!
Jake

**Comment by ivan-aksamentov** [comment]
Hi Jake @garfinjm,

Thanks for your kind words.

How QC works in general: we have a set of rules (4 currently) and each rule outputs a numeric score. Then the decision is made based on this score.

If you prefer to see the code, then this is how score is calculated in the "P" rule:

https://github.com/nextstrain/nextclade/blob/3621fdd0aaaa2f023f68791e19fba9fd36b6c85d/packages/web/src/algorithms/QC/rulePrivateMutations.ts#L19-L20

The `totalNumberOfMutations` here is the number of mutations that are in the sequence, but not in the closest reference tree node (what we call "private" mutations).

And then this is how the verdict is made based on score:
https://github.com/nextstrain/nextclade/blob/3621fdd0aaaa2f023f68791e19fba9fd36b6c85d/packages/web/src/algorithms/QC/QCRuleStatus.ts#L7-L14
Basically, below 30 is good, above 99 is bad and what's in the middle is mediocre.

You can set `typical` and `cutoff` parameters in the settings (click "Settings" button). Note that these settings will persist across runs and page reloads, and if you want to reset them back to defaults, there is a button.

Let me know if this answers your question. Otherwise we might ask Richard to elaborate. 

Do you think it is a reasonable definition for this rule? Will you do it differently? Let's discuss!

**Comment by rneher** [comment]
Hi @garfinjm,

the `private mutations` score is essentially the number of mutations by which your sequences differ from the closest sequence in our reference tree. You are right that our threshold is pretty stringent (10 mutations). But given the dense coverage of SARSCoV2 diversity, our experience at nextstrain is that most sequences have close matches and a strain with 10 mutations that we haven't seen before is something we look at more closely.  But in less well sampled lineages 10 private mutations can certainly happen and the flag for private mutations doesn't necessarily mean that the sequence is bad.

We could ignore mutations at the very 5' or 3' ends because they are masked in many downstream analysis anyway...
